test/inductor/test_cpu_repro.py

```diff
@@ -2237,8 +2190,10 @@ def fn(a):
         metrics.reset()
         x = torch.randn(1, 384, 20, 20).to(memory_format=torch.channels_last)
         self.common(fn, (x,))
-        assert metrics.generated_cpp_vec_kernel_count == 1
+        if codecache.valid_vec_isa_list():
+            assert metrics.generated_cpp_vec_kernel_count == 1
```

Comment:
Can we wrap it with a function, e.g., `check_vec_kernel_count(1)`?

Reply:
You mean the if and the assert? Sure makes sense, given how often it is used. I just a) didn't want to change too much in this PR and b) fear that with such a function the test is much less clear and e.g. `metrics.reset()` makes less sense on inspection and might be missed.

Reply:
Perhaps we can name it in a way to reminder the users, e.g., `check_metrics_vec_kernel_count`.

Reply:
Done. I also replaced other uses of `assert metrics.generated_cpp_vec_kernel_count ==` by a call to that function